verifact/forum/schema.py

```diff
@@ -39,3 +40,87 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+# MUTATIONS
+
+
+class CreateQuestion(Mutation):
```

Comment:
I prefer the reverse `QuestionCreate` because then Question mutations all get sorted together when using GraphiQL to browser the schema. What do you think?

Reply:
yep makes sense